tracing-log/src/lib.rs

```diff
@@ -221,6 +221,27 @@ impl AsTrace for log::Level {
 
 /// A simple "logger" that converts all log records into `tracing` `Event`s,
 /// with an optional level filter.
+///
+/// Be careful to set appropriate log levels:
+///
+/// * `LogTracer`'s default filter level is `log::LevelFilter::Info`.
+/// * `log`'s default max level is `log::LevelFilter::Off` (logging disabled).
```

Comment:
I don't know if this is actually the case by default, see: https://docs.rs/log/0.4.7/log/#compile-time-filters

Reply:
In the previous section (I think the default runtime filter is to skip logs even if it is not clearly documented.) it says:

Reply:
Ah, I see. In that case, perhaps we should have a helper function that both sets the `LogTracer` as the default logger _and_ sets the default log level at the same time, so that users don't have to remember to do both? I'd be happy to merge a follow-up branch that adds that...